pkg/registry/apis/provisioning/repository/go-git/wrapper.go

```diff
@@ -45,30 +45,10 @@ func init() {
 
 var _ repository.Repository = (*GoGitRepo)(nil)
 
-type GoGitCloneOptions struct {
```

Comment:
moved them to repository to use them for a more generic clonable interface concept